Makefile

```diff
@@ -0,0 +1,17 @@
+URL?=unknown
+SAUCE_USERNAME?=unknown
+SAUCE_ACCESS_KEY?=unknown
+
+# Node module variables
+MOCHA=./node_modules/.bin/mocha
+MOCHA_PARALLEL=./node_modules/.bin/mocha-parallel-tests
+
+
+# Concurrent test
+sauce:
+	@env URL=$(URL) SAUCE_USERNAME=$(SAUCE_USERNAME) SAUCE_ACCESS_KEY=$(SAUCE_ACCESS_KEY) $(MOCHA_PARALLEL) test/
+
+
+# Incremental test
+isauce:
```

Comment:
This should be a .PHONY target (https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)